browser_tests/menu.spec.ts

```diff
@@ -391,6 +394,17 @@ test.describe('Menu', () => {
       await tab.open()
     })
 
+    test.afterEach(async ({ comfyPage }) => {
```

Comment:
Instead of cleaning up the workflow, we can do a directory setup as well. See https://github.com/Comfy-Org/ComfyUI_frontend/blob/38e3dcbaebb97d311aa899aa77eca6d1fa5d8432/browser_tests/menu.spec.ts#L408-L411

Basically you can cleanup the workflows directory as an empty dir before as the test setup.

Reply:
I am also fine with the current solution, if you think the directory setup is too dangerous before we have a restoration mechanism for local run. Let me know which way you prefer.

Reply:
Okay I can do that and remove the temp names. 

When I use it locally I get

```
Error: Failed to setup workflows directory: 405: Method Not Allowed
```

Reply:
The devtools route is created by the Dev Tools custom node?

Reply:
Need https://github.com/Comfy-Org/ComfyUI_devtools/pull/2